tests/integration/ceph_base_file_test.go

```diff
@@ -435,6 +435,10 @@ func cleanupFilesystemConsumer(helper *clients.TestClient, k8sh *utils.K8sHelper
 	}
 	err = helper.FSClient.DeletePVC(namespace, podName)
 	assertNoErrorUnlessNotFound(s, err)
+	Isdeleted := k8sh.WaitUntilPVCIsDeleted(namespace, podName)
```

Comment:
nit: capital letters to begin the var name are usually only for package vars
```suggestion
	deleted := k8sh.WaitUntilPVCIsDeleted(namespace, podName)
```

Reply:
done